### DISCUSSION TITLE
Copying source member to another system without that member

### DISCUSSION THREAD

**Discussion by m-tyler** [codefori/vscode-ibmi#903] [IBM i]
Is there an easier way (without using git), to copy a source member from one system to another?

The steps I have used so far.
1. Find the member to download.
2. Using the right-click menu, select `Download` and save to a local drive.
3. Connect to the other system.
4. Expand the filter to that source file location on the other system.
5. Using the right-click menu, select Create Member.
6. Close opened member.
7. Using right-click menu on the member, select `Upload and replace`.
8. Open the member and do what you need to compile on other system.

-Matt

**Comment by andreas-prouza** [comment]
Hi Matt,
You can copy the content of the member to your clipboard (CTLR + A to mark the whole content then CTLR + C to copy).
On your other system create a meber and just past the content to it CTLR + V.
So you don't need to download and upload and open/close the member.

Greetings
Andy

**Comment by m-tyler** [comment]
Sorry, I should have been more clear.  I do know I can use copy and paste but I was hoping to see if there was a simplified method for copying the exact source (including the same source dates and line numbers) from one system to another.    Your method and my method require changing to another connection to put the new source. 

In RDI for those that have iSphere tools installed, there is a menu option,`Copy Member(s) to...` that performs essentially a SNDNETF for members.  

I could create an action for SNDNETF (and may still do this) but currently, I don't have the ability to send members using SNDNETF.

**Comment by andreas-prouza** [comment]
OK, I guess the plugin in RDi download the member local and upload it from local to the remote system.
At least I don't know such a funktion in vscode.
Escpacially you need to have some kind of input field to ask for the remote system.
As far as I know you can do that trick also in vscode but I think to copy a single member, your way is easier than to set up the enviroment to achieve this.
Also GIT would make much more effort to get a single member transferred.
But maybe someone else has a cool idea!

**Comment by m-tyler** [comment]
Thanks, for the feedback.

I was hoping to find additional options to allow me to stop using RDi.
